pkg/kubernetes/informers.go

```diff
@@ -171,6 +171,20 @@ func (im *InformerManager) AddPodListener(
 	})
 }
 
+// AddVolumeAttachmentListener hooks up add, update, delete callbacks.
+func (im *InformerManager) AddVolumeAttachmentListener(
+	add func(obj interface{}), update func(oldObj, newObj interface{}), remove func(obj interface{})) {
+	if im.volumeAttachmentInformer == nil {
+		im.volumeAttachmentInformer = im.informerFactory.Storage().V1().VolumeAttachments().Informer()
+	}
+
+	im.volumeAttachmentInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    add,
+		UpdateFunc: update,
```

Comment:
Can you remove this.

Reply:
I have removed this and manually tested it, but I see in many cases having functions that might not be used. Example: AddPVCListener, update and remove functions are nil.